labelbox/client.py

```diff
@@ -648,7 +651,9 @@ def get_data_row_metadata_ontology(self) -> DataRowMetadataOntology:
             DataRowMetadataOntology: The ontology for Data Row Metadata for an organization
 
         """
-        return DataRowMetadataOntology(self)
+        if self.data_row_metadata_ontology is None:
```

Comment:
The client doesn't cache any other resources. It only exposes get and create functions. I think at the least, we should make the data_row_metadata_ontology attribute private by prepending an underscore so that users don't directly access the attribute.

Although I kind of feel like the DataRowMetadataOntology class should be a singleton. The client already does too much here. I'll leave that up to you if you want to change it.